packages/interfaces/src/index.ts

```diff
@@ -50,6 +51,22 @@ export interface LightPush extends PointToPointProtocol {
   ) => Promise<SendResult>;
 }
 
+export interface PeerExchange extends PointToPointProtocol {
+  query(params: PeerExchangeQueryParams): Promise<PeerExchangeResponse>;
+}
+
+export interface PeerExchangeQueryParams {
+  numPeers: bigint;
```

Comment:
a `number` would be preferable

Reply:
Definitely. Will address this once the follow-up PRs are merged sequentially to avoid any divergent branches.

Reply:
addressed in the same branch